_includes/download-pdf.html

```diff
@@ -0,0 +1,7 @@
+{% if page.pdf.size > 0 %}
+  {% assign pdf_url = page.pdf %}
+  <div class="download-pdf">
+    <a href="{{ page.pdf }}" download="{{ page.title }}" title="{{ site.data.language.download_pdf }}"><i class="icon-download"></i>
+    <div class="download-pdf__text">{{ site.data.language.download_pdf }}</div></a>
```

Comment:
Since this is an inline element, it should be a `<span>` tag instead of the div. There should also be two of them, one for the mobile language "PDF" and one for the desktop language "Download PDF". You can use CSS to determine which one is visible based on screen size.